Src/ILGPU/Runtime/ExchangeBuffer.cs

```diff
@@ -313,6 +314,42 @@ public void CopyFromAccelerator(AcceleratorStream stream)
             return buffer.Buffer;
         }
 
+        /// <summary>
```

Comment:
MINOR: Please move all the functionality from the `Operators` region to the `Methods` region.